src/api/app/models/package.rb

```diff
@@ -349,6 +349,28 @@ def kiwi_file_md5
     nil
   end
 
+  def changes_files
+    result = []
+    dir_hash.elements('entry') do |e|
+      result << e['name'] if e['name'] =~ /.changes$/
+    end
+    result
+  end
```

Comment:
Then let's reach out to the backend team and ask for an api endpoint that returns all changes files of a package.

Or do you have another solution in mind?